docs/csharp/language-reference/keywords/default-values-table.md

```diff
@@ -46,10 +34,34 @@ Remember that using uninitialized variables in C# is not allowed.
 |[ulong](ulong.md)|0|
 |[ushort](ushort.md)|0|
 
+## Remarks
+
+Use [default value expression](../../programming-guide/statements-expressions-operators/default-value-expressions.md) to produce the default value of a type as the following example shows:
+
+```csharp
+int a = default(int);
+```
+
+Beginning with C# 7.1, you can use the [`default` literal](../../programming-guide/statements-expressions-operators/default-value-expressions.md#default-literal-and-type-inference) to initialize a variable with the default value of its type:
+
+```csharp
+int a = default;
+```
+
+You also can use the default constructor to produce the default value of a value type:
```

Comment:
This is unrelated to your PR, but I'd prefer the following change:

default constructor --> default constructor or the implicit default constructor

The problem here is that the primitives don't actually have constructors; they're an artifact of the compiler, which replaces the constructor call with a call to the IL Initobj instruction. If you want, you can cross-reference the [constructors topic](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constructors), which mentions both default and implicit default constructors, or an [old blog post on implicit default constructors](https://blogs.msdn.microsoft.com/bclteam/2010/04/28/constructors-and-value-types-ron-petrusha/).